spec/units/user_spec.rb

```diff
@@ -12,4 +12,12 @@
       expect(user.id).to eq persisted_data[0]["id"]
     end
   end
+  describe ".authentificate" do
+    it "checks the user exists" do
+      User.create(username: "test_user", email: "test@test.com", password: "test_password")
+      auth = User.authentificate(email: "test@test.com", password: "test_password")
+      p auth
+      expect(auth.email).to eq "test@test.com"
+    end
+  end
```

Comment:
You could add another test to give yourself some reassurance that the method returns nil. 

You can also remove the p statement, and perhaps make the 'auth' variable more descriptive so that it is clearer (perhaps 'return_value'?)

Reply:
new test added